doc/source/rllib/rllib-training.rst

```diff
@@ -1237,12 +1238,45 @@ do:
 The level of parallelism within the evaluation step is determined via the ``evaluation_num_workers``
 setting. Set this to larger values if you want the desired evaluation episodes or timesteps to
 run as much in parallel as possible. For example, if your ``evaluation_duration=10``,
-``evaluation_duration_unit=episodes``, and ``evaluation_num_workers=10``, each eval RolloutWorker
+``evaluation_duration_unit=episodes``, and ``evaluation_num_workers=10``, each RolloutWorker
 only has to run 1 episode in each evaluation step.
 
+In case you observe occasional failures in your RolloutWorkers during evaluation (e.g. you have an environment that sometimes crashes),
+you can use an (experimental) new setting: ``enable_async_evaluation=True``.
+This will run the parallel sampling of all evaluation RolloutWorkers via a fault tolerant, asynchronous manager, such that if one of the workers
+takes too long to run through an episode and return data or fails entirely, the other RolloutWorkers will pick up its task and complete the job.
```

Comment:
Do the other workers pick the job up because of recreate_failed_workers or because of this setting? I don't quite understand what happens if I for example set `enable_async_evaluation=True` and `ignore_worker_failures=True`, Would it still retry the failed episode? or would it just return one less trajectory than specified? Same thing for `enable_async_evaluation=True` and `recreate_failed_workers=True`? 
What happens if all three are true? is there any error checking for this not to happen?

Reply:
It was already mentioned above, where we explain these two settings in more detail. Added a link to that section.